core/dbt/parser/base.py

```diff
@@ -348,16 +350,21 @@ def update_parsed_node(
         values set in _build_intermediate_parsed_node.
         """
         config_dict = config.config
+
+        # Set tags on node provided in config blocks
+        model_tags = config_dict.get('tags', [])
+        parsed_node.tags.extend(model_tags)
+
+        # do this once before we parse the node schema/alias, so
+        # parsed_node.config is what it would be if they did nothing
+        self.update_parsed_node_config(parsed_node, config_dict)
```

Comment:
there's a duplicate call to 
```
 self.update_parsed_node_config(parsed_node, config_dict)
```

at the end of this method. Do we need/want that?

Reply:
Nope!